compiler/frontend/src/org/jetbrains/kotlin/diagnostics/Errors.java

```diff
@@ -1243,6 +1243,8 @@ enum BadNamedArgumentsTarget {
             DiagnosticFactory1.create(WARNING, PositioningStrategies.USELESS_ELVIS);
     DiagnosticFactory0<KtBinaryExpression> USELESS_ELVIS_RIGHT_IS_NULL =
             DiagnosticFactory0.create(WARNING, PositioningStrategies.USELESS_ELVIS);
+    DiagnosticFactory0<KtBinaryExpression> USELESS_ELVIS_LEFT_IS_NULL =
```

Comment:
Please don't modify K1 sources (`compiler/frontend`), it is no longer developed.